libs/wingsdk/src/ex/react-app.md

```diff
@@ -0,0 +1,87 @@
+---
+title: React Website
+id: react-website
+description: A built-in resource for creating deployable websites using the React framework.
+keywords: [Website, React, deployment, build]
+sidebar_position: 1
+---
+
+The `ex.ReactApp` resource represents a website, built using React, that can be both hosted in the cloud or to run on a development hot-reload server in the sim target.
+
+## Usage
+
+### Initialization
+
+Using the default arguments:
+
+```ts
+bring ex;
+bring util;
+
+let website = new ex.ReactApp(projectPath: "./client");
+```
+
+or customizing them:
+
+```ts
+bring ex;
+bring util;
+
+let website = new ex.ReactApp(
+  projectPath: "./client",
+  isDevRun: false // `true`` by default. will run the start command if true, and the build command if not
+  buildDir: "/dist" // default is "/build"
+  startCommand: "pnpm start" // default is "npm start"
+  buildCommand: "pnpm build" // default is "npm build"
+  localPort: 4000 // default is 3001
+ );
+```
+
+When `ReactApp` is compiled to the `sim` target, by default it runs the start command (default: `npm start`) inside `projectPath` to serve your app in development mode on a local port.
+
+If the `CI` environment variable is set OR if `ReactApp` is compiled to any other target, it will run the build command (default: `npm build`) inside of the `projectPath` to build the React app for production to `buildDir` and serve the app.
+
```

Comment:
It doesn't look like this was implemented-

If we're keeping `isDevRun` then let's update the docs to make sure it's clear that the option only has an effect on the `sim` target, and that the `buildCommand` is always used for building the react app on other targets.